### ISSUE TITLE
Multiblock turbine bug

### ISSUE THREAD

**Issue by BigotteClement** [mekanism/Mekanism#6016]
Please use the search functionality before reporting an issue. Also take a look at the closed issues!

Issue description:
Hello all,
I try to make a turbine on my personnal server. I have try ... to many times to say you exactly how many ^^ I have follow a lot of tuto and i don't think i have made errors on my build. When i put the last bloc of my structure i don't have the validation animation so i can't use the turbine. All possible structure on this mod works perfectly only this one..
I have check on a new solo world with the latest version and same problems. Maybe i do something wrong i don't know.

![image](https://user-images.githubusercontent.com/33891984/78064329-e228d380-7391-11ea-864e-23e2224b6fd1.png)
![image](https://user-images.githubusercontent.com/33891984/78064363-ee149580-7391-11ea-822b-78b75fb7c342.png)
![image](https://user-images.githubusercontent.com/33891984/78064399-fc62b180-7391-11ea-89f1-4af710153636.png)

Steps to reproduce:
Version (make sure you are on the latest version before reporting):
Forge: 31.1.25
Mekanism: 9.9.16.408
Other relevant version: 1.15.2

**Comment by BigotteClement** [comment]
![Sans titre](https://user-images.githubusercontent.com/33891984/78065267-50ba6100-7393-11ea-94e2-fb7335c10380.png)
![Sans titre2](https://user-images.githubusercontent.com/33891984/78065268-51eb8e00-7393-11ea-99d9-982a001d8d4d.png)
![Sans titre3](https://user-images.githubusercontent.com/33891984/78065270-51eb8e00-7393-11ea-861c-c9b14bfb336f.png)
 
Sorry. I reupload the screenshots.

**Comment by thiakil** [comment]
I'm not seeing any vent blocks there

**Comment by BigotteClement** [comment]
They're around the Electromagnetic Coil.
But i think the problem is here .. all the tuto i have seen, the electromagnetic coil are connected but has you can see in the picture they're not connected..

**Comment by BigotteClement** [comment]
Around the Rotational Complex*

**Comment by BigotteClement** [comment]
If someone can create a map in 1.15.2 and the same mekanism version and send me the map, it will be cool maybe ?

**Comment by thiakil** [comment]
No, vents are a different thing, they just look similar to the pressure dispersers.
They need to be on the outside layer - please recheck your the tutorials you've looked at

Connected textures doesn't have anything to do with functionality, check you have the CTM mod installed

**Comment by BigotteClement** [comment]
![Sans titre4](https://user-images.githubusercontent.com/33891984/78081661-5f177580-73b1-11ea-87f0-e5f649c31aa5.png)
![Sans titre5](https://user-images.githubusercontent.com/33891984/78081662-5fb00c00-73b1-11ea-8879-b243b072bd30.png)


 I know .. like that ?
Ok i doesn't know this mod

**Comment by BigotteClement** [comment]
Ok my bad ..
I confused vents and pressure dispersers.
Thks a lot bro.
